src/test/tests/databases/export_db.py

```diff
@@ -126,15 +134,107 @@ def VTK_check_binary(filename):
     b = "BINARY" == line[0:6]
     return b
 
-def cleanup_files():
-    exts = ("*.ply", "*.raw", "*.stl", "*.silo", "*.tec", "*.okc", "*.visit", "*.xyz", "*.obj")
-    files = os.listdir(".")
-    for f in files:
-        for e in exts:
-            pattern = e[1:]
-            if f[-len(pattern):] == pattern:
-                os.unlink(f)
-                break
+
+# Test exporting some surfaces in other formats and plotting the results.
+# e = ExportDBAttributes
+# v = View3D
+# a = AnnotationAttributes
+# count = current test number (used for test names)
+# testNum = 1 or 2 (for test1 or test2 which tests write groups)
+
+def export_and_plot(e, v, a, count, testNum):
+
+    # testNum ==1 no write groups
+    # testNum ==2 using write groups
+    # formats fields:
+    # {DBName: (no-write-group exported name to read, write-group exported name to read, plot type, var name)}
+
+    formats = {
+"PLY":          ("export_PLY.ply",          "*.ply",         "Subset",      "PLY_mesh"),
```

Comment:
Is the odd indentation here an attempt to minimize max column width?

Reply:
It was a holdover from original code. I have made a change, is it better now?

Reply:
Yes, better. Thanks. Was probably me...it looks like something I would do 🤣